core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1549,67 +1550,68 @@ private void deleteInternal(RpcContext rpcContext, LockedInodePath inodePath, bo
       // Alluxio URIs which could not be deleted
       List<String> failedUris = new ArrayList<>();
 
-      TempInodePathForDescendant tempInodePath = new TempInodePathForDescendant(inodePath);
-      // We go through each inode, removing it from its parent set and from mDelInodes. If it's a
-      // file, we deal with the checkpoints and blocks as well.
-      for (int i = delInodes.size() - 1; i >= 0; i--) {
-        Pair<AlluxioURI, Inode> delInodePair = delInodes.get(i);
-        AlluxioURI alluxioUriToDel = delInodePair.getFirst();
-        Inode delInode = delInodePair.getSecond();
+      try (TempInodePathForDescendant tempInodePath = new TempInodePathForDescendant(inodePath)) {
```

Comment:
Is this just an indenting change?

Reply:
I added a try  resource block around tempInodePath creation, hence the indenting change.